### ISSUE TITLE
You can get clip file path from media pool

### ISSUE THREAD

**Issue by fukco** [deric/DaVinciResolve-metadata#3]
Nice Job, Dude. Actually Resolve API provide this, try to use [MediaPoolItem]:GetClipProperty("File Path")

**Comment by deric** [comment]
I'm not sure what's the issue here. Try to submit it as a pull request.

**Comment by fukco** [comment]
Sorry, it's not a bug seriously. you said it can not get media pool item's file path, so you add a media folder select button, i mean you can delete this button and read media pool clips by default

**Comment by deric** [comment]
You're right, I see. I had issues with some API calls that weren't working (at least on Linux). But I don't remember details, I'll try to have a look at it. Thanks for the hint!

**Comment by fukco** [comment]
I did something like this repo,  but not use exiftool. i use python to call my dynamic link library write by go to do this. the parser only support sony XAVC file so far. I tested  [MediaPoolItem]:GetClipProperty("File Path") on Windows & MacOS :joy:
Here is my python script:https://github.com/fukco/DaVinciResolveScript/blob/main/Metadata%20Parser.py
Maybe something is helpful
README is write by Chinese :laughing: code is english :grin:
